resources/strings/src/commonMain/kotlin/dev/sasikanth/rss/reader/resources/strings/DeTwineStrings.kt

```diff
@@ -0,0 +1,149 @@
+        settingsOpmlCancel = "Stornieren",
+        settingsPostsDeletionPeriodTitle = "Gelesene Beiträge löschen",
+        settingsPostsDeletionPeriodOneWeek = "1 Woche",
+        settingsPostsDeletionPeriodOneMonth = "1 Monat",
+        settingsPostsDeletionPeriodThreeMonths = "3 Monate",
+        settingsPostsDeletionPeriodSixMonths = "6 Monate",
+        settingsPostsDeletionPeriodOneYear = "1 Jahr",
+        settingsShowReaderViewTitle = "Enable reader view",
+        settingsShowReaderViewSubtitle = "Articles will be shown in a simplified, easy-to-read view",
+        feeds = "Einspeisungen",
+        editFeeds = "Feeds bearbeiten",
+        comments = "Kommentare",
+        about = "Um",
+        aboutRoleDeveloper = "Entwickler",
+        aboutRoleDesigner = "Designerr",
+        aboutSocialTwitter = "Twitter",
+        aboutSocialThreads = "Threads",
+        aboutSocialGitHub = "GitHub",
+        aboutSocialWebsite = "Webseite",
+        feedsSearchHint = "Such-Feeds",
+        allFeeds = "Alle Feeds",
+        pinnedFeeds = "Angepinnt",
+        openWebsite = "Website öffnen",
+        markAllAsRead = "Alles als gelesen markieren",
+        noNewPosts = "Keine neuen Inhalte",
+        noNewPostsSubtitle =
+        "Schauen Sie später noch einmal vorbei oder klicken Sie nach unten, um jetzt nach neuen Inhalten zu suchen",
+        postsAll = "Alle",
+        postsUnread = "Ungelesen",
+        postsToday = "Heute",
+        openSource = "Open Source",
+        openSourceDesc =
+        "Twine basiert auf Open-Source-Technologien und ist völlig kostenlos. Den Quellcode von Twine und einigen meiner anderen beliebten Projekte finden Sie auf GitHub. Klicken Sie hier, um dorthin zu gelangen.",
+        markAsRead = "Als gelesen markieren",
+        markAsUnRead = "Als ungelesen markieren",
+        removeFeed = "Futter entfernen",
+        delete = "Löschen",
+        removeFeedDesc = { "Möchten Sie entfernen? \"${it}\"?" },
+        alwaysFetchSourceArticle = "Vollständige Artikel immer in der Leseransicht abrufen",
+        getFeedInfo = "Informationen bekommen",
+        newTag = "Neuer Tag",
+        tags = "Stichworte",
+        addTagTitle = "Tag hinzufügen",
+        tagNameHint = "Name",
+        tagSaveButton = "Speichern",
+        deleteTagTitle = "Tag löschen?",
+        deleteTagDesc =
+        "Das Tag wird gelöscht und aus allen zugewiesenen Feeds entfernt. Ihre Feeds werden nicht gelöscht",
+        feedOptionShare = "Aktie",
+        feedOptionWebsite = "Webseite",
+        feedOptionRemove = "Entfernen",
+        feedTitleHint = "Titel",
+        noUnreadPostsInFeed = "Keine ungelesenen Artikel",
+        numberOfUnreadPostsInFeed = { numberOfUnreadPosts ->
+            when (numberOfUnreadPosts) {
+                1L -> "$numberOfUnreadPosts ungelesener Artikel"
+                else -> "$numberOfUnreadPosts ungelesener Artikel"
+            }
+        }
+    )
+
```

Comment:
1L -> "$numberOfUnreadPosts ungelesener Beitrag"
else -> "$numberOfUnreadPosts ungelesene Beiträge"

Reply:
thanks for the cross check

Reply:
can you kindly resolve the PR